Sources/SwiftGenKit/Parsers/InterfaceBuilder/Scene.swift

```diff
@@ -70,11 +74,16 @@ func == (lhs: InterfaceBuilder.Scene, rhs: InterfaceBuilder.Scene) -> Bool {
   return lhs.identifier == rhs.identifier &&
     lhs.tag == rhs.tag &&
     lhs.customClass == rhs.customClass &&
-    lhs.customModule == rhs.customModule
+    lhs.customModule == rhs.customModule &&
+    lhs.moduleIsPlaceholder == rhs.moduleIsPlaceholder &&
+    lhs.platform == rhs.platform
 }
 
 extension InterfaceBuilder.Scene: Hashable {
   var hashValue: Int {
-    return identifier.hashValue ^ tag.hashValue ^ (customModule?.hashValue ?? 0) ^ (customClass?.hashValue ?? 0)
+    let part1: Int = identifier.hashValue ^ tag.hashValue
+    let part2: Int = (customModule?.hashValue ?? 0) ^ (customClass?.hashValue ?? 0)
+    let part3: Int = moduleIsPlaceholder.hashValue ^ platform.hashValue
+    return part1 ^ part2 ^ part3
```

Comment:
Use the new `hash(into:)` instead?

Reply:
I wanted to but it’s swift 4.2 only.

Reply:
Oh. Thought it was introduced earlier than that.

Maybe a `// FIXME:` for later then

Reply:
That'll trigger a swiftlint warning though? Which will cause CI to fail 😆
(unless we add a `swiftlint:disable:next`)

Reply:
Ah! Damn. Ok so maybe create a "Migrate the Codebase to Swift 4.2" dedicated issue and note that in there to not forget.